Setup/Model/ProductProtection/ProductProtectionV1.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/*
+ * Copyright Extend (c) 2023. All rights reserved.
+ * See Extend-COPYING.txt for license details.
+ */
+
+namespace Extend\Integration\Setup\Model\ProductProtection;
+
+use Exception;
+use Extend\Integration\Service\Extend;
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Catalog\Model\Product\Attribute\Source\Status;
+use Magento\Catalog\Model\Product\Type;
+use Magento\Catalog\Model\Product\Visibility;
+use Magento\Catalog\Model\Product;
+use Magento\Catalog\Model\ProductFactory;
+use Magento\Eav\Api\Data\AttributeSetInterface;
+use Extend\Integration\Setup\Model\AttributeSetInstaller;
+use Magento\Store\Model\StoreManagerInterface;
+
+// NOTE: make sure to update ProductInstaller's use statement with this class if it is the current version
+class ProductProtectionV1 implements ProductProtectionInterface
+{
+    const VERSION = 'V1';
```

Comment:
Let's use something other than "V1" since this is used to reference Guidance already.